pkg/restore/split.go

```diff
@@ -272,7 +271,7 @@ func getSplitKeys(rewriteRules *RewriteRules, ranges []rtree.Range, regions []*R
 		checkKeys = append(checkKeys, rule.GetNewKeyPrefix())
 	}
 	for _, rg := range ranges {
-		checkKeys = append(checkKeys, truncateRowKey(rg.EndKey))
+		checkKeys = append(checkKeys, rg.EndKey)
```

Comment:
Why do we add `truncateRowKey` in the first place? @5kbpers

Reply:
Originally we may append a suffix like `0xff` at the end of table range, need to truncate it here for getting a valid key. 🤔